src/main/java/core/basesyntax/strategy/Operation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface Operation {

```

Comment:
```suggestion
public interface OperationHandler {
```
use descriptive names, Operation is just an operation